index.js

```diff
@@ -223,19 +224,48 @@ async function main() {
       core.debug(inspect(data));
     }
   
-    if (inputs.reviewers) {
-      core.debug(`Adding reviewers: ${inputs.reviewers}`);
-      const reviewers = inputs.reviewers.trim().split(/\s*,\s*/);
+    if (inputs.reviewers || inputs.team_reviewers) {
+      let params = {
+        owner,
+        repo,
+        pull_number: number
+      }
+      let reviewers = null;
+      let team_reviewers = null;
+
+      if(inputs.reviewers) { 
+        core.debug(`Adding reviewers: ${inputs.reviewers}`) 
+        reviewers = (inputs.reviewers ?? "").trim().split(/\s*,\s*/);
```

Comment:
I think the Nullish coalescing operator is not available in Node 12. But we can update the runner to node16 which is available now

https://github.com/gr2m/create-or-update-pull-request-action/blob/3a2f600a564e7f2cb6e2ac9671bb77439ca7628e/action.yml#L57

Reply:
Ah apologies, I was just touching up the commit and commenting after I cleaned it up to lyk, but i got you

Reply:
Have that upgrade in this same PR?

Reply:
yes, either change it to `"node16"` or don't use the nullish coalescing operator

Reply:
I'll just change it to `||`